### PR TITLE
Yield disbursement

### PR DESCRIPTION
**Pull request by RonTuretzky**
Motivation: 
Forward yield to a dedicated contract for the disbursement of yield 

Steps: 

- [x] Add an address holder for the yield handling contract
- [x] Add a function to retrieve the yield 
- [x] Add a setter function for the yield handling contract

### CODE REVIEW

**Review by kassandraoftroy — APPROVED** [review]
LGTM

File: src/Bread.sol [hunk 1]
```diff
@@ -56,7 +58,9 @@ contract Bread is
         __ERC20_init(name_, symbol_);
         __Ownable_init(owner_);
     }
-
+    function setYieldDisburser(address _yieldDisburser) external onlyOwner {
+        yieldDisburser = _yieldDisburser;
```
Comment (kassandraoftroy) [inline src/Bread.sol]: Maybe add an event ?

File: src/Bread.sol [hunk 2]
```diff
@@ -89,6 +93,12 @@ contract Bread is
 
         emit ClaimedYield(amount);
     }
+    function claimYieldForDisbursement() external {
```
Comment (kassandraoftroy) [inline src/Bread.sol]: Wondering if having "both" of these yield claimers is problematic ? Should we only keep this new one ?

File: src/Bread.sol (hunk 1 above)
Comment (RonTuretzky) [reply src/Bread.sol]: added 👍👍

File: src/Bread.sol (hunk 2 above)
Comment (RonTuretzky) [reply src/Bread.sol]: added only an owner modifier to it, I think it's fine for us to have it in case of errors for now. later on should be removed.

**Comment by RonTuretzky** [comment]
@kassandraoftroy merge?

**Pull request merged**